### DISCUSSION TITLE
How can I export the results of my experiments as CSV?

### DISCUSSION THREAD

**Discussion by jameswpm** [langchain-ai/langsmith-sdk#912] [Q&A] [answered]
The following documentation page only includes instructions on how to download the examples. I want to export the results of the experiments.

https://docs.smith.langchain.com/how_to_guides/datasets/manage_datasets_in_application#export-a-dataset

A simple CSV containing the same information we have on the experiments table is enough (e.g. experiment label, error rate, metrics, etc...)

**Comment by hinthornw** [comment, marked as answer]
Could use this: https://github.com/langchain-ai/langsmith-cookbook/blob/main/testing-examples/export-test-to-csv/export-test-to-csv.ipynb
or the constituent APIs

**Reply by jameswpm** [reply]
This was precisely what I was looking for, but I needed help finding it searching around. Thanks for pointing me to the link. I'll test it to verify, but it looks fine.
